search-api/tests/unit/test_api.py

```diff
@@ -0,0 +1,191 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to verify the entities API end-point.
+
+Test-Suite to ensure that the /entities endpoint is working as expected.
+"""
+
+import copy
+import json
+from unittest.mock import patch
+
+from tests.utilities.factory_utils import factory_auth_header
+from tests.utilities.factory_scenarios import TestJwtClaims, JWT_HEADER
+from search_api import status as http_status
+
+
+def _dir_search(client, jwt, session, params):
+    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.no_role)
+
+    rv = client.get('/api/v1/directors/search/{}'.format(params),
+                    headers=headers, content_type='application/json')
+
+    assert rv.status_code == http_status.HTTP_200_OK
+    return json.loads(rv.data)
+
```

Comment:
Should have created Json schema for response and validate the response against the schema.

Reply:
json schema (as in jsonschema.net) or Marshmallow ?

Reply:
jsonschema.net.  
You can refer https://github.com/bcgov/sbc-pay/tree/development/pay-api/src/pay_api/schemas

https://github.com/bcgov/sbc-pay/blob/development/pay-api/tests/unit/api/test_fee.py#L66

Reply:
Ticketed here, https://github.com/bcgov/sbc-search/issues/125

Reply:
done